rpcserver.go

```diff
@@ -1090,6 +1091,11 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 		return nil, fmt.Errorf("cannot send coins to pubkeys")
 	}
 
+	label, err := labels.ValidateAPI(in.Label)
```

Comment:
Using this here is tricky as it can be a user initiated send (`lncli sendcoins`) and we'll label it as external if no label is provided, unless that's intended.

Reply:
It is intentional, because I think of external as covering any non-lnd initiated tx (user sending from cli, rpc call etc). We can't strictly draw a  user/application line between cli and rpc - something like lndmanage uses lncli on the user's behalf, a wallet that sends coins using the rpc is really just a user send etc, so I think `External` is a good coverall?